oidc-controller/api/authSessions/crud.py

```diff
@@ -11,26 +12,27 @@
     AuthSessionCreate,
     AuthSessionPatch,
 )
-from ..db.session import db, COLLECTION_NAMES
+from ..db.session import COLLECTION_NAMES
 
 
 logger = logging.getLogger(__name__)
 
 
 class AuthSessionCRUD:
-    @classmethod
```

Comment:
Is is expected for these annotations to disappear?

Reply:
yes, the crud class is no longer a 'singleton', the db context gets passed on initialization of the instance. this makes it better and easier for testing databases to be injected instead of the runtime db.